torch/functional.py

```diff
@@ -1262,7 +1262,12 @@ def norm(input, p="fro", dim=None, keepdim=False, out=None, dtype=None):  # noqa
         more consistent with NumPy's numpy.linalg.norm.
 
     Args:
-        input (Tensor): the input tensor
+        input (Tensor): The input tensor. The data type must be either floating
```

Comment:
"The data type must be either floating point or complex" -> "Its data type must be either a floating point or complex type."